Utils/azuretools-core/src/com/microsoft/azuretools/utils/CommandUtils.java

```diff
@@ -149,4 +177,47 @@ private static String getSafeWorkingDirectory() {
             return DEFAULT_MAC_LINUX_PATH;
         }
     }
+
+    public static class CommendExecOutput {
```

Comment:
`Commend` <-> `Command`

Reply:
is this class required? 
is it possible to use Exception to avoid adding a new class